packages/cli/test/integration/evaluation/test-runs.api.test.ts

```diff
@@ -0,0 +1,204 @@
+import { Container } from 'typedi';
+
+import type { TestDefinition } from '@/databases/entities/test-definition.ee';
+import type { User } from '@/databases/entities/user';
+import type { WorkflowEntity } from '@/databases/entities/workflow-entity';
+import { TestDefinitionRepository } from '@/databases/repositories/test-definition.repository.ee';
+import { TestRunRepository } from '@/databases/repositories/test-run.repository.ee';
+import { createUserShell } from '@test-integration/db/users';
+import { createWorkflow } from '@test-integration/db/workflows';
+import * as testDb from '@test-integration/test-db';
+import type { SuperAgentTest } from '@test-integration/types';
+import * as utils from '@test-integration/utils';
+
+let authOwnerAgent: SuperAgentTest;
+let workflowUnderTest: WorkflowEntity;
+let otherWorkflow: WorkflowEntity;
+let testDefinition: TestDefinition;
+let otherTestDefinition: TestDefinition;
+let ownerShell: User;
+
+const testServer = utils.setupTestServer({ endpointGroups: ['evaluation'] });
+
+beforeAll(async () => {
+	ownerShell = await createUserShell('global:owner');
```

Comment:
Would be nice to have tests for the scope based permissions as well. I.e. check that:
1. Someone who is not a global owner but has access to the workflow can access these
2. Someone who does not have access gets a Forbidden error

I think it would probably make sense that these type of tests were unit tests with mocked dependencies, instead of integration tests. But feel free to decide for yourself

Reply:
Good suggestion! Added one more integration test for the case 1, case 2 was already covered